src/Drivers/EloquentEntitySet.php

```diff
@@ -362,7 +361,11 @@ protected function getRelationships(): array
 
             $relations = array_merge(
                 $relations,
-                Arr::prependKeysWith($expansionSet->getRelationships(), "{$expansionPropertyName}.")
+                Collection::make($expansionSet->getRelationships())
+                    ->mapWithKeys(function ($item, $key) use ($expansionPropertyName) {
+                        return ["{$expansionPropertyName}.".$key => $item];
+                    })
+                    ->all()
```

Comment:
Hello. Can you tell me why this particular method of adding prefixes was chosen? It would be interesting to hear your opinion.